### PR TITLE
Create test_kubectl_delete.py

### PR DESCRIPTION
**Pull request by zeeshanali**
Create test_kubectl_delete.py

### CODE REVIEW

File: tests/integration/test_kubectl_delete.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+    logger.debug(
+        "Removing pod '%s' from model '%s' with cmd: %s", pod_name, ops_test.model_name, cmd
+    )
+
+    retcode, stdout, stderr = await ops_test.run(*cmd)
+    assert retcode == 0, f"kubectl failed: {(stderr or stdout).strip()}"
+    logger.debug(stdout)
+    await ops_test.model.block_until(lambda: len(ops_test.model.applications[app_name].units) > 0)
+    await ops_test.model.wait_for_idle(apps=[app_name], status="blocked", timeout=1000)
```
Comment (gruyaume) [inline tests/integration/test_kubectl_delete.py:58]: I'm a bit confused here, what are we validating? That a statefulset will re-spin pods after you've deleted them?

File: tests/integration/test_kubectl_delete.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
+# Copyright 2022 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+
```
Comment (gruyaume) [inline tests/integration/test_kubectl_delete.py:5]: In the PR message, can you please explain the reason behind this change?

**Review by gruyaume — CHANGES_REQUESTED** [review]

File: tests/integration/test_kubectl_delete.py (hunk 2 above)
Comment (zeeshanali) [reply tests/integration/test_kubectl_delete.py:5]: actually as part of my OB i have to add test coverage in pytest i asked Chris and he said ok i can add kubcectl delete as test coverage in this code as part of my Onboarding at canonical

Comment (zeeshanali) [reply tests/integration/test_kubectl_delete.py:5]: You mean in License i have no idea honestly i copied the file from other charm and leave the license part as it is

File: tests/integration/test_kubectl_delete.py (hunk 1 above)
Comment (zeeshanali) [reply tests/integration/test_kubectl_delete.py:58]: yes , we dont need to test that part , the whole idea is if we can delete the pods after they are created ..

File: tests/integration/test_kubectl_delete.py (hunk 2 above)
Comment (gruyaume) [reply tests/integration/test_kubectl_delete.py:5]: This test is an integration test so it won't make a difference in code coverage and I don't want to add a test just for the sake of it, we'll have to maintain them once they're added. That being said, if we find places where there's a behaviour that isn't validated in the current tests, yes let's add tests to validate it.

Comment (zeeshanali) [reply tests/integration/test_kubectl_delete.py:5]: actually i was missing the test to delete the pods like in other integration tests . can we figure out any solution to add above ?

**Pull request closed without merging**